doc/api/esm.md

```diff
@@ -722,6 +722,19 @@ const instance2 = await WebAssembly.instantiate(libraryModule, {
 });
 ```
 
+In addition to the static source phase, there is also a dynamic variant of the
+source phase via the `import.source` dynamic phase import syntax:
+
+<!-- eslint-skip -->
```

Comment:
Let's amend the linter config so it supports `import.source` instead of disabling it
````diff
diff --git a/eslint.config.mjs b/eslint.config.mjs
index 777524660d..c866337997 100644
--- a/eslint.config.mjs
+++ b/eslint.config.mjs
@@ -102,6 +102,7 @@ export default [
       parser: babelEslintParser,
       parserOptions: {
         babelOptions: {
+          parserOpts: { createImportExpressions: true },
           plugins: [
             babelPluginSyntaxImportAttributes,
             babelPluginSyntaxImportSource,
````

Reply:
I tried adding this locally but it didn't seem to fix the linting case for `import.source()`. Let's perhaps figure this out as a follow-up rather.

Reply:
Actually that was another linting error so this did work fine, NVM! Should be resolved now thanks.